### ISSUE TITLE
Default theme for Xoops and Newbb

### ISSUE THREAD

**Issue by cesagonchu** [XoopsModules25x/newbb#121]
Several problems:

1) Facebook UI Style Rating is only visible with the Xoops default theme

2) With default theme, the define THEME_FORUM_REPLY is not translated

3) With default theme, the Quick Reply window won't close

![newbbthemedefault](https://user-images.githubusercontent.com/4613970/112752151-e81bb980-8fd1-11eb-9bdd-bccd34a82d16.jpg)

___
My config:

PHP 7.4.9
MYSQL 5.7.33-0ubuntu0.16.04.1
XOOPS 2.5.11-Beta2 (today's version)
Newbb 5.01 Beta 4 (today's version)

**Comment by cesagonchu** [comment]
The problem still exists with the default theme.
Instead of the cross we now have a button named "&times;" and a window that invades the forum message and does not close.

![image](https://user-images.githubusercontent.com/4613970/128817124-f85f9319-2564-47e1-afda-cf83d67c0247.png)

**Comment by mambax7** [comment]
How many people are still using the default theme? I'm not sure if anybody is still using it, and thus justify the time investment needed for it.

**Comment by cesagonchu** [comment]
Yes, I have been thinking for a long time that we should remove it from the core. It would scare away a new user :-P

**Comment by mambax7** [comment]
Well, it is good for developers for testing, but the Admin should not give it as an option to a user, that's what we have xBootstrap or xSwatch themes  :)
